lib/membrane_vp8.ex

```diff
@@ -3,5 +3,12 @@ defmodule Membrane.VP8 do
   This module implements struct describing a VP8 encoded video stream.
   """
 
-  defstruct []
+  @type t :: %__MODULE__{
+          width: non_neg_integer | nil,
+          height: non_neg_integer | nil,
+          rate: non_neg_integer | nil,
+          scale: non_neg_integer | nil
```

Comment:
I think these params should be required or they shouldn't be passed through caps...